docs/eigenda/operator-guides/requirements/delegation-requirements.mdx

```diff
@@ -40,9 +40,9 @@ Details about how these requirements are enforced and the process by which DA no
 
 ## Checking eligibility
 
-In order to determine the current TVL of the top 200 operators, please visit our
+In order to determine the current TVL of the top 200 operators for each quorum, please visit our
 AVS page ([Mainnet](https://app.eigenlayer.xyz/avs/eigenda), [Holesky](https://holesky.eigenlayer.xyz/avs/eigenda)) and sort by `TVL
-Ascending.` Observe the first 200 operators listed and the amount of ETH TVL
+Ascending.` Observe the first 200 operators listed for the quorum and the amount of ETH TVL
 delegated to them. Please keep in mind that the AVS Page reflects the
 operator stake on EigenLayer, which is used to update the EigenDA operator set
 stake weights on a weekly basis (Wednesdays at 17:00 UTC), so the EigenDA stake
```

Comment:
Is weekly the current update cycle?

Reply:
Current update cycle is 6 hours, and updating this was an action item from the ops review today.
It will be done in a separate PR